.github/dependabot.yml

```diff
@@ -10,3 +10,8 @@ updates:
   schedule:
     interval: weekly
   open-pull-requests-limit: 99
+- package-ecosystem: enums
+  directory: "/enums"
```

Comment:
Why do we need to specify it? We don't specify rust-code-analysis-cli and rust-code-analysis-web, but they are picked up by Dependabot anyways

Reply:
Because we have excluded it: https://github.com/mozilla/rust-code-analysis/blob/master/Cargo.toml#L39